recipes/argo-cd.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: argo
+    url: https://argoproj.github.io/argo-helm
+    type: helm
+  name: argo-cd
```

Comment:
Let's not use `name` fields since we have deprecated it 
```suggestion
```

Reply:
done